src/test/regress/sql/multi_mx_modifications_to_reference_tables.sql

```diff
@@ -0,0 +1,91 @@
+
+
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1470000;
```

Comment:
Minor: We generally shouldn't rely on the shard ID sequence. If the output contains shard IDs we should use `SET citus.next_shard_id TO 1470000`.